sentry/values.yaml

```diff
@@ -179,6 +179,9 @@ sentry:
     nodeSelector: {}
     # tolerations: []
     # podLabels: []
+    # customCA:
```

Comment:
I would add customCA value to sentry level and use it everywhere from there. Usually if you need custom CA for web you will need it for worker as well. Also deployment-sentry-cron needs the same changes, it calls relay api and as far as I discovered deployment-sentry-ingest-metrics-consumer-perf and deployment-sentry-ingest-metrics-consumer-rh have ssl errors if you need customCA